src/App/__snapshots__/app.test.js.snap

```diff
@@ -9,21 +9,11 @@ exports[`App tests should render default App component 1`] = `
   >
     <Alert
       action={
-        <Button
-          aria-label={null}
-          className="dismiss-notification"
-          component="button"
-          isActive={false}
-          isBlock={false}
-          isDisabled={false}
-          isFocus={false}
-          isHover={false}
-          onClick={[Function]}
-          type="button"
-          variant="secondary"
```

Comment:
@cdcabrera Good call - In this case it looks like just using the AlertActionCloseButton fixes the problem by forcing you to use their supplied button. I will look into the page issue you brought up.